server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -219,8 +217,9 @@ private String convertToString(ArrayList<String> comments) {
 
     for (String comment : comments) {
       comment = comment.replace("\"", "");
-      
-      if (comment.length() - 1 <= 0){
+      // comment.replace() could potentially give an empty string,
```

Comment:
This comment just tells me how replace() and substring() work, and doesn't tell me why a comment with a length of 0 or 1 shouldn't be modified below (that comments with 0 or 1 characters aren't considered sentences)